src/radio/index.js

```diff
@@ -282,16 +290,18 @@ class Radio extends EventEmitter {
     }
 
     if (this.order.length > 0) {
+      // $FlowFixMe
       const index = this.order.findIndex(u => this.queues.has(u.id) && this.queues.get(u.id).length > 0);
       if (index !== -1) {
         const user = this.order[index];
-        const queue = this.queues.get(user.id);
+        const queue = this.queues.get(user.id) || [];
```

Comment:
This still feels really bad because the fallback case makes things explode anyway.

Maybe we'll solve this better later.